### PR TITLE
Adding README.md in other languages 

### PR DESCRIPTION
**Pull request by KerinPithawala**
### Description

Thanks for contributing this Pull Request. Make sure that you send in this Pull Request to the `develop` branch of this repository, add a brief description, and tag the relevant issue(s) and PR(s) below.

- Relevant Issues :  Solves issue #16 

- What Changed and Why? : Added workflow in .github folder to translate README.md file into different languages. As a result of this action, the languages added in README.md are: Chinese Traditional, Hindi, Arabic and French

- Type of change :
  - [ ] New feature
  - [ ] Bug fix for existing feature
  - [ ] Code quality improvement
  - [ ] Addition or Improvement of tests
  - [x] Addition or Improvement of documentation

- Other:
  - [ ] Add unit tests
  - [x] Add documentation

### CODE REVIEW

**Comment by KerinPithawala** [comment]
Please review my PR for Grace Hopper Celebration Open Source Day.

File: .github/workflows/readme.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+name: Translate README
+
+on:
+  push:
+    branches:
+      - main
+      - master
```
Comment (katielevy1) [inline .github/workflows/readme.yml:7]: Let's remove the reference to master here as we only have a "main" branch and want to be inclusive

**Review by katielevy1 — CHANGES_REQUESTED** [review]
This is so awesome, so many languages! Can we update the English readme to have links to each of these readme links so people can find them easily?

File: .github/workflows/readme.yml (hunk 1 above)
Comment (KerinPithawala) [reply .github/workflows/readme.yml:7]: Yes, we can. I am making the required changes. Thank you for the feedback.

**Comment by KerinPithawala** [comment]
I have removed the master branch on the actions. Please review my PR. 
Also, could you please tell me where it is suitable to add other README.md linked in the main README.md file?

**Review by katielevy1 — APPROVED** [review]
Looks great!

**Comment by katielevy1** [comment]
@all-contributors add @KerinPithawala for documentation

**Pull request merged**